docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -283,3 +283,54 @@ GET /transactions/{tx_id}
 
    :statuscode 200: A transaction with that ID was found.
    :statuscode 404: A transaction with that ID was not found.
+
+
+GET /unspents/
+-------------------------
+
+.. http:get:: /unspents?owner_after={owner_after}
+
+   Get a list of links to transactions' conditions that have not been used in
+   a previous transaction and could hence be called unspent conditions/outputs
+   (or simply: unspents).
+
+   This endpoint doesn't return anything if the querystring ``owner_after``
+   happens to not be defined in the request.
+
+   Note that if unspents for a certain ``owner_after`` have not been found by
+   the server, this will result in the server returning a 200 OK HTTP status
+   code and an empty list in the response's body.
+
+   :param owner_after: A public key, able to validly spend an output of a
+   transaction, assuming the user also has the corresponding private key.
+   :type owner_after: hex string
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      GET /unspents?owner_after=1AAAbbb...ccc HTTP/1.1
+      Host: example.com
+
+   **Example response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      [
+        {
```

Comment:
The decision to return a list of JSON `TransactionLink`s here was rationalized with the following argument:

When creating a `TRANSFER`, the creator is interested both in the transaction's id and the position of the condition (`cid`), but also in the actual condition to be fulfilled/spent.

`txid` and `cid` need to be included as `input` in `fulfillments[x].fulfillment`.
To create `fulfillments[x].fulfillment.fulfillment`, the original condition solves as the basis of the fulfillment.